brave_paks.gni

```diff
@@ -61,6 +61,7 @@ template("brave_extra_paks") {
       "$root_gen_dir/brave/browser/resources/settings/brave_settings_resources.pak",
       "$root_gen_dir/components/brave_components_resources.pak",
       "$root_gen_dir/brave/components/brave_ads/resources/brave_ads_resources.pak",
+      "$root_gen_dir/bat/ads/resources/bat_ads_resources.pak",
```

Comment:
@bridiver Unless I add this resource pak the service fails to start, is this correct or should I have approached this differently?

Reply:
this is not correct, `bat_ads_resources.pak` is already part of brave_ads_resources

Reply:
what is the actual error?